docs/low-code-sql/development/upload-sql-files.md

```diff
@@ -0,0 +1,96 @@
+---
```

Comment:
I would have put this file as a subpage of this existing page: 
docs/low-code-sql/development/gems/datasources.md

If you want to do that, here are the steps:
1. create this path: `docs/low-code-sql/development/gems/datasources/`
2. move datasources.md and upload-sql-files.md into this ^ path.
3. create a new file `_category_.json` on this ^ path 
3a. set `"label": "datasources",` 
4. move the *.png files used in datasources.md and upload-sql-files into a new directory `docs/low-code-sql/development/gems/datasources/img`

Reply:
Done.

Had to set the label to Data Sources.